### DISCUSSION TITLE
.zip file type's association with osu!

### DISCUSSION THREAD

**Discussion by cheeezie** [ppy/osu#21796] [Q&A]
While opening zip files in my Linux system, it gives me an option to open it in osu. What are zip files used for in this context? I thought there were only .osk files and its other variations (p.s I've never seen such effect on Windows or Android before from my experience)
![Screenshot_20221226_114850](https://user-images.githubusercontent.com/102518950/209527529-52ccbf0e-0bf4-4da6-94aa-d81bd1ce04ba.png)

**Comment by peppy** [comment]
No idea, this should not be the case.
